src/btree/bt_sync_obsolete.c

```diff
@@ -338,7 +338,7 @@ __checkpoint_cleanup_page_skip(
      * checks to verify the cache can handle it.
      */
```

Comment:
Can we expand the comment here?
```suggestion
     * checks to verify the cache can handle it. Checkpoint cleanup leads to more clean pages in the cache, skip if clean eviction is needed.
     */
```